dagql/idtui/testdata/TestTelemetry/TestGolden/cached-execs

```diff
@@ -0,0 +1,30 @@
+Expected stderr:
+
+✔ connect X.Xs
+│ ✔ starting engine X.Xs
+│ ✔ connecting to engine X.Xs
+│ ✔ starting session X.Xs
+
+✔ initialize X.Xs
+│ ✔ resolving module ref X.Xs
+│ ✔ installing module X.Xs
+│ ✔ analyzing module X.Xs
+
+✔ prepare X.Xs
+
+✔ viztest: Viztest! X.Xs
+✔ .cachedExecs: Void X.Xs
+│ ✔ container: Container! X.Xs
+│ $ .from(address: "alpine"): Container! X.Xs CACHED
+│ │ ✔ resolving docker.io/library/alpine:latest X.Xs
+│ ✔ .withExec(args: ["echo", "cached-execs cached for good"]): Container! X.Xs
+│ ┃ cached-execs cached for good
```

Comment:
Why doesn't this get cached? :eyes: Is that the flake?

Reply:
Yep 😭 - see this thread: https://github.com/dagger/dagger/pull/8442#discussion_r1765037152

I could commit the non-flaked output but it'd be a Sisyphean task to keep it from just getting re-set to the broken output once someone runs `dagger call test telemetry --update -o .` and commits the result.